### PR TITLE
adding HTCondor prerelease test recipe

### PR DESCRIPTION
**Pull request by osg-cat**

### CODE REVIEW

File: docs/release/condor-itb-testing.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+1. Shut down HTCondor and HTCondor-CE on prerelease machines:
+
+        :::console
+        ansible condordev -i inventory -bK -f 20 -m service -a 'name=condor-ce state=stopped' -l 'itb-ce*'
+        ansible condordev -i inventory -bK -f 20 -m service -a 'name=condor state=stopped'
+
+2. Install new version of HTCondor on prerelease machines:
+
+        :::console
+        ansible condordev -i inventory -bK -f 10 -m command -a 'yum --enablerepo=osg-upcoming --assumeyes update condor'
```
Comment (brianhlin) [inline docs/release/condor-itb-testing.md]: Shouldn't we be installing from `osg-upcoming-development`? We should also think about specifying the version of condor we want installed

File: docs/release/condor-itb-testing.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+    or, if you need to install an NVR that is “earlier” (in the RPM sense) than what is currently installed:
+
+        :::console
+        ansible condordev -i inventory -bK -f 10 -m command -a 'yum --enablerepo=osg-development --assumeyes downgrade condor condor-classads condor-python condor-procd blahp'
```
Comment (brianhlin) [inline docs/release/condor-itb-testing.md]: We should think about specifying the condor version here as well

File: docs/release/condor-itb-testing.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+## Prerequisites
+
+The following items are known prerequisites to using this recipe.  If you are not running the Ansible commands from
+osghost, there are almost certainly other prerequisites that are not listed below.  And even using osghost for Ansible
+and itb-submit for the submissions, there may be other prerequisites missing.  Please improve this document by adding
+other prerequisites as they are identified!
+
+* A checkout of the osgitb directory from our local git instance (not GitHub)
+* Your X.509 DN in the `osgitb/unmanaged/htcondor/condor_mapfile` file and (via Ansible) on `itb-ce1` and `itb-ce2`
```
[the remark this answers is not in the document]
Comment (osg-cat) [reply docs/release/condor-itb-testing.md]: Yes, of course, but that does not affect this version of the document.

File: docs/release/condor-itb-testing.md (hunk 1 above)
Comment (osg-cat) [reply docs/release/condor-itb-testing.md]: Well, it depends on what @timtheisen wants tested… In this particular case, he asked to have HTCondor 8.6.6 tested, and it was in regular development.

File: docs/release/condor-itb-testing.md (hunk 2 above)
Comment (osg-cat) [reply docs/release/condor-itb-testing.md]: Well, if necessary. My goal was not to create examples of every variation on doing the HTCondor installs. I figure that you all use yum enough to sort it out.

**Comment by osg-cat** [comment]
I have commented on all of your review comments. At the moment, I see no changes to make, but if you like, go ahead and make whatever changes you see fit. I would like to get this published today, so that we can check off the “test HTCondor prereleases on ITB” item, finally.

File: docs/release/condor-itb-testing.md (hunk 1 above)
Comment (brianhlin) [reply docs/release/condor-itb-testing.md]: Then shouldn't it read `osg-development` instead of `osg-upcoming`?

Comment (osg-cat) [reply docs/release/condor-itb-testing.md]: Ah, shoot, copy-and-paste fail from my bash history. I will go fix that.

**Pull request merged**